### ISSUE TITLE
(Placement Group): (Setting placement group name does not work)

### ISSUE THREAD

**Issue by NdemeraS** [aws/aws-cdk#28361]
### Describe the bug

Unable to manually set/change the name of a placement group using with the [L1](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.CfnPlacementGroup.html#attrgroupname) or [L2](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.PlacementGroup.html#placementgroupname-1) construct 

### Expected Behavior

As the the feature is documented, the expectation is to able to manually set a placement group name

### Current Behavior

The CDK will auto-generate a name for the placement group regardless if you is manually set

### Reproduction Steps

```
import { Duration, Stack, StackProps } from 'aws-cdk-lib';
import * as ec2 from 'aws-cdk-lib/aws-ec2';
import { Construct } from 'constructs';

export class PlacementStack extends Stack {
  constructor(scope: Construct, id: string, props?: StackProps) {
    super(scope, id, props);

    const placem = new ec2.PlacementGroup(this, 'MyPlacementGroup', /* all optional props */ {
    partitions: 1,
    placementGroupName: 'Testing2341413',
//    spreadLevel: ec2.PlacementGroupSpreadLevel.HOST,
    strategy: ec2.PlacementGroupStrategy.PARTITION,
    });
  }
}

```

### Additional Information/Context

The underlying [CFN placement group](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-placementgroup.html) resource has no support for the placement group name property. Since the CDK is a wrapper for the CFN, there may be some back end integration that is not working as expected

### CDK CLI Version

2.108.1 (build 2320255)

### Node.js Version

v18.17.1

### OS

n/a

### Language

TypeScript

**Comment by pahud** [comment]
Yes. As you mentioned, the placegroup name is not a property in its CFN resource and if we look at the constructor from CDK source code, it does not really pass to the L1 resource.

https://github.com/aws/aws-cdk/blob/b8b6bafc07b0804f6e802ba563ff3b7deaea6eee/packages/aws-cdk-lib/aws-ec2/lib/placement-group.ts#L194-L204

I guess maybe the L2 resource can create a `Name` tag with the placegroup name as its value but it's very opinionated and requires some discussion.

**Comment by pahud** [comment]
OK I can confirm this should be a gap from cloudformation and it's not possible to specify the Name through CFN and CDK cannot do anything with that.

Please create a feature request in [cloudformation-coverage-roadmap](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues) github repo and allow the community to vote for more visibility. After CFN supports that, CDK would follow up and support it very soon.
